src/main/java/jaz/Zer.java

```diff
@@ -50,10 +51,33 @@
                Iterator, Runnable, Callable, Function, Collection, List {
   static final long serialVersionUID = 42L;
 
+  private static final Map<String, Set<String>> HOOKS_USING_JAZ_ZER = new HashMap<String,
+      Set<String>>() {
+    {
+      put("com.code_intelligence.jazzer.sanitizers.ReflectiveCall",
+          new HashSet<>(
+              Arrays.asList("java.lang.Class.forName", "java.lang.ClassLoader.loadClass")));
+
+      put("com.code_intelligence.jazzer.sanitizers.Deserialization",
+          new HashSet<>(Arrays.asList("java.io.ObjectInputStream.readObject",
+              "java.io.ObjectInputStream.readObjectOverride",
+              "java.io.ObjectInputStream.readUnshared")));
+
+      put("com.code_intelligence.jazzer.sanitizers.ExpressionLanguageInjection",
+          new HashSet<>(Arrays.asList("javax.el.ExpressionFactory.createValueExpression",
+              "javax.el.ExpressionFactory.createMethodExpression",
+              "jakarta.el.ExpressionFactory.createValueExpression",
+              "jakarta.el.ExpressionFactory.createMethodExpression",
+              "javax.validation.ConstraintValidatorContext.buildConstraintViolationWithTemplate")));
```

Comment:
Given the extent of these manual lists, I think that we should add test coverage for all the three sanitizers. Maybe it's possible to reuse the existing Java test code and just add new Bazel test targets with the appropriate changes (set the property and modify `allowed_findings`)?